install/packages.rst

```diff
@@ -184,6 +184,22 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
+::
+
+  robotpy-installer download-opkg -r requirements.txt
+
+This file matches the syntax of a standard pip `requirements.txt`.
```

Comment:
Ehh… pip's requirements.txt is a fair bit more complex than what we have here (notably no version constraints or nesting of requirements files). Unless if you plan to fix that in the near future, we can just say it's a list of packages.

Reply:
Changed the wording of it a bit, should be a lot more accurate now.